cfg/sources.js

```diff
@@ -3,7 +3,8 @@ export const ATTRIBUTION_CHANNEL = Object.freeze({
     LINKEDIN_DM: 'LINKEDIN_DM', 
     LINKEDIN_POST: 'LINKEDIN_POST', 
     TRELLUS_WEBSITE: 'TRELLUS_WEBSITE',
-    MANUAL: 'MANUAL'
+    MANUAL: 'MANUAL',
+    CALLBLITZ: 'CALLBLITZ',
```

Comment:
lgtm, but does this need to be added to db as an enum? i forget how we chose to implement